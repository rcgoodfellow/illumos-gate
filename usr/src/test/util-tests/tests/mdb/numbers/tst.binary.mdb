0i100_0111=K
0i11_101=j
0i1__1__0______=X
0t2$d
100_0001__0001=K
1_____________0=K
