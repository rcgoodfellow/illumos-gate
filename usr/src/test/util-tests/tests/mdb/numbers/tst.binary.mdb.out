                47              
                11101
                ||| |
                ||| +-- bit 0 mask 0x01
                ||+---- bit 2 mask 0x04
                |+----- bit 3 mask 0x08
                +------ bit 4 mask 0x10
                6               
radix = 2 base ten
                411             
                2               
